src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,109 @@
+.cards-container {
```

Comment:
each blocke should be in separate file.
move styles for container in other file.